.semaphore/semaphore.yml

```diff
@@ -162,7 +162,7 @@ blocks:
     task:
       agent:
         machine:
-          type: s1-prod-ubuntu20-04-arm64-1
+          type: s1-prod-ubuntu20-04-arm64-2
```

Comment:
Are you sure this is based off of the latest `main`? [This line in main](https://github.com/confluentinc/ide-sidecar/blob/08595b7f594d11eca31be9e95a3331dcfe38aad5/.semaphore/semaphore.yml#L165) is already changed to use `-2`.